From edf2b0c0574f65cef0c54af73a1932d34aaed6bd Mon Sep 17 00:00:00 2001
From: Heinrich Schuchardt <xypron.glpk@gmx.de>
Date: Sun, 7 Apr 2019 13:41:50 +0200
Subject: [PATCH 1/1] test: env: Enable env unit tests by default

If CONFIG_UNIT_TEST is enabled we should enable the individual tests by
default to ensure good test coverage.

Signed-off-by: Heinrich Schuchardt <xypron.glpk@gmx.de>
---
 test/env/Kconfig | 1 +
 1 file changed, 1 insertion(+)

diff --git a/test/env/Kconfig b/test/env/Kconfig
index ff164132e9..6cb82337b3 100644
--- a/test/env/Kconfig
+++ b/test/env/Kconfig
@@ -1,6 +1,7 @@
 config UT_ENV
 	bool "Enable env unit tests"
 	depends on UNIT_TEST
+	default y
 	help
 	  This enables the 'ut env' command which runs a series of unit
 	  tests on the env code.
-- 
2.20.1

